client/chromez-api.js

```diff
@@ -24,3 +24,31 @@ function removeCard(card) {
   if (cardview)
     return Polymer.Base.async(function() { cardview.removeCard(card); });
 }
+
+function addDash(dashSpec) {
+  if (typeof dashSpec == "object")
+  {
+    var dash = dashSpec.name;
+    var key = dashSpec.key;
+  } else if (dashSpec.indexOf("(") > -1) {
+    var bits = dashSpec.split("(");
+    var dash = bits[0];
+    var key = bits[1].split(")")[0];
+  } else {
+    var dash = dashSpec;
+    var key = undefined;
+  }
+
+  var e = dashpoint.querySelector(dash);
+  if (e && e.key == key)
+    return;
+  Polymer.Base.importHref(dash + '.html',
+    function(e) {
+      var elt = document.createElement(dash);
+      elt.key = key;
+      dashpoint.insertBefore(elt, feed);
```

Comment:
we should probably require a container to add the dashes to, rather than require that there's an element with id=feed